src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/ArrayComparisonOperator.java

```diff
@@ -3,7 +3,9 @@
 /** List of element level operator that can be used in Filter clause */
 public enum ArrayComparisonOperator implements FilterOperator {
   ALL("$all"),
-  SIZE("$size");
+  SIZE("$size"),
+  // Will not be supported from outside
+  NOTANY("$notany");
```

Comment:
This is used only for negation for $all, not exposed as a filter operator

Reply:
Maybe add this exact explanation in Java comment, as that is very helpful.